### PR TITLE
🎨 Add brand colors and fonts to README.md

### LINKED ISSUE #4: Add nebari colors' hex values to this repo
### Context

It'll be helpful to have the nebari brand colors shared here for quick access. :)

Nebari colors:
- Yellow: #EAB54E
- Green: #20AAA1
- Black: #0F1015
- Purple: #BA18DA

### Value and/or benefit

Besides the logos, folks may also like to use the brand colors for diagrams, presentations, etc. when they refer to Nebari.

### Anything else?

Maybe we can share this list in the README itself?

### PR DESCRIPTION
**Pull request by pavithraes**
## Reference Issues or PRs

Fixes: #4

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ You can find the assets in the following locations:
...
+The Nebari brand uses the following primary colors:
+
+- Yellow: #EAB54E
+- Green: #20AAA1
+- Purple: #BA18DA
+- Black: #0F1015
+
+And, the following fonts:
+
+- Atkinson Hyperlegible
```
Comment (trallard) [inline README.md]: Suggestion: add links to the fonts in Google Fonts

**Review by trallard — CHANGES_REQUESTED** [review]
Only one comment re links for fonts and should be ready for merge

File: README.md (hunk 1 above)
Comment (pavithraes) [reply README.md]: That's a good idea, done!

**Comment by trallard** [comment]
Thank you, this LGTM 🚀  merging now!

**Pull request merged**